os/tools/mksyscall.c

```diff
@@ -543,10 +543,12 @@ int main(int argc, char **argv, char **envp)
 		case '?':
 			fprintf(stderr, "Unrecognized option: %c\n", optopt);
 			show_usage(argv[0]);
+			break;
 
 		case ':':
 			fprintf(stderr, "Missing option argument, option: %c\n", optopt);
 			show_usage(argv[0]);
+			break;
 
 		default:
```

Comment:
I think we will not reach default at all as the above cases cover all possibilities, if so can we remove the default case?

Reply:
I think we need to keep this because of unexpected inputs.